doc/example/conversion_reaction/PEtabJl_module.jl

```diff
@@ -5,15 +5,15 @@ using Sundials
 using PEtab
 
 pathYaml = "/Users/pauljonasjost/Documents/GitHub_Folders/pyPESTO/test/julia/../../doc/example/conversion_reaction/conversion_reaction.yaml"
-petabModel = readPEtabModel(pathYaml, verbose=true)
+petabModel = PEtabModel(pathYaml, verbose=true)
 
 # A full list of options for createPEtabODEProblem can be found at https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.setupPEtabODEProblem
```

Comment:
Change to PEtabODEProblem in the coment (createPEtabODEProblem no longer exists)